File: 0229-majority-element-ii/0229-majority-element-ii.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        a=Counter(nums)
        ans=[]
        Threshold=len(nums)//3
        for i,j in a.items():
            if j>Threshold:
                ans.append(i)
        return ans

File: 0229-majority-element-ii/0229-majority-element-ii.cpp
class Solution {
public:
    vector<int> majorityElement(vector<int>& nums) {
        vector<int>v;
        unordered_map<int,int>h;
        for(int i:nums){
            h[i]++;
        }
        int thre=(nums.size()/3);
        for(auto& [key,value]:h){
            if(value>thre){

                 v.push_back(key);
            }
        }
        return v;
    }
};